_rules/image-filename-as-accessible-name-9eb3f6.md

```diff
@@ -61,45 +68,88 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-This `img` element has an [accessible name][] equivalent to the filename. The [accessible name][] accurately describes the image.
+This `img` element has an [accessible name][] equivalent to the filename (ignoring letter casing). The [accessible name][] accurately describes the image.
 
 ```html
 <html lang="en">
-	<img src="https://www.w3.org/WAI/demos/bad/img/w3c" alt="w3c" />
+	<img src="/test-assets/image-filename-as-accessible-name-9eb3f6/nyhavn" alt="Nyhavn" />
 </html>
 ```
 
 #### Passed Example 2
 
-This `img` element has an [accessible name][] equivalent to the filename. The [accessible name][] in combination with the text content of the `a` element accurately describes the image.
+This `img` element has an [accessible name][] equivalent to the filename. Because the image is used in a download link, the presence of the file extension is a relevant part of its description.
 
 ```html
 <html lang="en">
-	<a href="https://www.w3.org/WAI/demos/bad/img/w3c.png" download
-		>Download <img src="https://www.w3.org/WAI/demos/bad/img/w3c.png" alt="w3c.png"
+	<a href="/test-assets/image-filename-as-accessible-name-9eb3f6/nyhavn.jpeg" download
+		>Download <img src="/test-assets/image-filename-as-accessible-name-9eb3f6/nyhavn.jpeg" alt="nyhavn.jpeg"
 	/></a>
 </html>
 ```
 
+#### Passed Example 3
+
+This image button has an [accessible name][] equivalent to the filename. The [accessible name][] accurately describes the purpose of the button.
+
+```html
+<html lang="en">
+	<input type="image" src="test-assets/image-filename-as-accessible-name-9eb3f6/login" alt="login" />
+</html>
+```
+
+#### Passed Example 4
```

Comment:
This example seems unnecessary. Why did you add this?

Reply:
I think we need a passed example with an image button, given that the rule is targeting them.